easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-5.9-gompi-2021b-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'OSU-Micro-Benchmarks'
+version = '5.9'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://mvapich.cse.ohio-state.edu/benchmarks/'
+description = """OSU Micro-Benchmarks"""
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://mvapich.cse.ohio-state.edu/download/mvapich/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['d619740a1c2cc7c02a9763931546b320d0fa4093c415ff3873c2958e121c0609']
+
+dependencies = [
+    ('CUDA', '11.4.1', '', True),
+    ('NCCL', '2.10.3', versionsuffix),
+    ('OpenMPI-CUDA', '4.1.1', versionsuffix),
+    ('UCX-CUDA', '1.11.2', versionsuffix),
```

Comment:
This one is not strictly needed since OpenMPI-CUDA includes it.

Reply:
done in c6e310a